### DISCUSSION TITLE
关于PCL2本体问题导致的游戏崩溃，各位的看法是什么

### DISCUSSION THREAD

**Discussion by HomoobyDX** [Meloong-Git/PCL#5317] [讨论]
这两天注意到HMCL汉医堂出现多个PCL本体导致的崩溃，PCL2本体问题总不能让他以后都换成HMCL吧，那PCL2的用户就都被HMCL抢过去了
所以来征求一下大家的意见
如果我看到有PCL2本体导致的，我会尽量把错误报告附过来

**Comment by MoYuan-CN** [comment]
先说一句
PCL 本体崩溃很少发生，很大情况下是 MC 和 PCL 因为内存溢出而同时崩溃
PCL 本体应当是无法导致 MC 本体崩溃的

**Reply by HomoobyDX** [reply]
我打算以后遇到PCL本体问题的就转过来，然后交给社区解答，这样可以尽量规避HMCL要去修PCL的bug（bushi）

**Comment by 3gf8jv4dv** [comment]
> 这两天注意到HMCL汉医堂出现多个PCL本体导致的崩溃

可以简单解释下具体的情况吗？或者给个简单的上下文（如交流记录截图）

**Reply by HomoobyDX** [reply]
![QQ截图20241221233104](https://github.com/user-attachments/assets/2434e70e-57f2-4c81-80e1-6eb4865735f1)

**Reply by HomoobyDX** [reply]
差不多是这种情况，这两天我关注一下群里出现PCL本体导致的错误，然后上传到这里供社区参考

**Comment by Viola-Siemens** [comment]
我简单说一下情况，之前我也赞助过龙猫但没加 PCL 用户群，我本人也是希望启动器多元化，让用户有更多的选择。不过对于 PCL 目前的现状存在一些作为启动器可选功能上的空白，如**无法辅助调试**（如未响应时没有提供方法导出游戏运行栈、显卡不可用时**无法切换 OpenGL 外的其它渲染器**（如软渲染、Vulkan 渲染器等）、**无法增/删/替换模组加载器或 OptiFine** 等。另外偶尔还会出现版本 json 格式异常或损坏等情况，也是 PCL 可能存在的问题，这些等我在 HMCL 崩溃交流群里遇到之后尽快发在这里反馈。

**Reply by LTCatt** [reply]
如果遇到 PCL 导致的问题，在这里反馈之后我就会处理，在群里说我也不知道的，多谢理解，多谢理解 Orz

**Comment by Viola-Siemens** [comment]
[错误报告-2024-12-27_21.06.18.zip](https://github.com/user-attachments/files/18260565/-2024-12-27_21.06.18.zip)
启动参数有点问题

**Reply by LTCatt** [reply]
可以在 https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+Bug%2C%E6%96%B0%E6%8F%90%E4%BA%A4&projects=&template=bug1.yml 交
